### DISCUSSION TITLE
W1130 10:59:06.035883   35871 loader.go:223] Config not found: /etc/kubernetes/admin.conf

### DISCUSSION THREAD

**Discussion by ZonzeeLi** [k3s-io/k3s#6577] [Q&A] [answered]
Download binary files and image files. Execute the following commands on the master node to complete the installation.

```bash
INSTALL_K3S_SKIP_DOWNLOAD=true ./install.sh
```

But I got the node information and said I was missing the file

[root@k8s-master ~]# kubectl get node
W1130 10:59:06.035883   35871 loader.go:223] Config not found: /etc/kubernetes/admin.conf
The connection to the server localhost:8080 was refused - did you specify the right host or port?

**Comment by brandond** [comment, marked as answer]
Please see https://docs.k3s.io/cluster-access
